### ISSUE TITLE
Defaulting plushu user to SUDO_USER when run with sudo

### ISSUE THREAD

**Issue by stuartpb** [plushu/plushu#68]
So, I just tried setting this up on AWS, which uses the default Ubuntu configuration where `root` is locked down and there's an `ubuntu` user. Running this to install Plusku as root doesn't work, because we default `PLUSHU_USER` to _the user running sudo_ when running as root:

```
sudo plushu plugins:install plusku
```

You have to _manually state that you mean to manipulate plushu_ which is kind of stupid (It's less likely that you want to manipulate Plushu for your own account):

```
sudo PLUSHU_USER=plushu plushu plugins:install plusku
```

Also, the "no .plushurc" rejection is broken because it tries to source a non-existent `profile.sh` before loading the plushurc... we should probably just use the file relative to `BASH_SOURCE` or whatever when doing that call rather than `PLUSHU_ROOT` (or just use that as the default if only `PLUSHU_USER` is defined).

Basically a whole bunch of the default user profile stuff needs to be rethought.

**Comment by stuartpb** [comment]
Actually, thinking about how this could be used in a real shared environment... I think the best solution would be to use the current user (sudo or not) _only if they have their own .plushurc_, and then _otherwise_ fall back to `plushu`. It's all a bit "more magic", but it's also magic that _really matches the behavior you expect for all non-pathological cases_ (and it's magic you can still override with `PLUSHU_USER`).

**Comment by stuartpb** [comment]
As for profile.sh... yeah, I think we're just going to have to settle for vendoring it inline in the `plushu` script, with maybe a test in CI that detects if the two have gone out of sync. It's not pretty or elegant, but it's the reality of how `plushu` is structured.
